### ISSUE TITLE
Getting Mutable Data keys

### ISSUE THREAD

**Issue by josephm1** [maidsafe/beaker-plugin-safe-app#114]
MD keys are returned as an object rather than uint8array like md values. This means developers are not able to use .toString() on keys but are able to use it on values. There is a work around for this 
```javascript 
function uintToString(key) {
	let uintArray = new Uint8Array(Object.values(key));
	return new TextDecoder('utf-8').decode(uintArray);
}
``` 
(taken from [Feedy](https://github.com/josephm1/Feedy/blob/master/main.js))
which returns the md keys as a string. Also I noticed that as a string the letter "r" was added on to one of the keys in the latest browser.

**Comment by bochaco** [comment]
Thanks @josephm1 , I'm looking into it now.

**Comment by bochaco** [comment]
This is being solved with PR #115.
